readme.html

```diff
@@ -129,20 +251,21 @@ <h3>Test Connectivity</h3>
 <a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_new_test_connectivity.png">
     <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_new_test_connectivity.png">
 </a>
-<h3>Interactive Messages</h3>
-In order for Phantom to receive responses to questions from Slack, the functionality needs to be configured. But first, you will need to create a user to allow Slack to authenticate with Phantom (This user is separate from the automation user created above).
+<h2>Set up Socket Mode in Slack</h2>
+Go to the <b>Your apps</b> option in Slack. From the menu on the left select the <b>Socket Mode</b>
+option.
 <br><br>
-Navigate back to the <b>Users</b> tab in <b>Administration</b>, and click on <b>+ USER</b>.
-<br><br>
-<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_users.png">
-    <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_users.png">
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_socket_mode.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_socket_mode.png">
 </a>
 <br><br>
-In the pop-up window, select the User Type <b>Local</b>. Fill out the required username and password fields. For security reasons DO NOT give the user any roles. This user should only be used to authenticate POST requests from Slack to Phantom.
+Once on this page, toggle on <b>Socket Mode</b>. Then click on the the event subscription option. This will redirect you to the Event Subscription page, and add the following subscriptions for bot.
```

Comment:
```suggestion
Once on this page, toggle on <b>Socket Mode</b>. Then click on the event subscription option. This will redirect you to the Event Subscription page, and add the following subscriptions for bot.
```

Reply:
Done